src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,6 +1,31 @@
 import React from 'react';
 import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+import { Post } from '../types/Post';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+export const PostInfo: React.FC<Post> = ({
+  body,
+  id,
+  title,
+  user,
+  comments,
+}) => (
+  <div
+    className="post__element"
+    key={id}
+    data-cy="post-info"
+  >
+    <h1 className="post__title" data-cy="post-title">
+      {title}
+    </h1>
+    {user
+      ? <UserInfo name={user.name} email={user.email} />
+      : ''}
```

Comment:
This looks simpler, agree?
```suggestion
    {user && (
      <UserInfo name={user.name} email={user.email} />)}
```

Reply:
Let's use such approach